examples/index.html

```diff
@@ -254,20 +254,6 @@ <h2 class="text-2xl font-bold text-blue-600">
         </div>
       </div>
 
-      <div class="card">
```

Comment:
Seems like this was already removed in #1026 
Why is this diff re-appearing?

Reply:
I branched off #1026 but github shouldn't pick it up since it got merged 🤔